README.rst

```diff
@@ -24,6 +24,13 @@ configurable email templates, grouping of notifications to ease management and s
 
 ------------
 
+Available Features
+------------------
+
+* Generation and transmission of notifications
+* Receive notifications on email
+* Customizable notifications
```

Comment:
I'd write `Configurable and extensible notification types` and would link a new section where we explain how to register / unregister notification types.